Makefile

```diff
@@ -199,7 +199,6 @@ init-coverage: ## Initialize converage report for Cilium unit-tests.
 unit-tests: start-kvstores ## Runs all unit-tests for Cilium.
 	$(QUIET) $(MAKE) $(SUBMAKEOPTS) -C tools/maptool/
 	$(QUIET) $(MAKE) $(SUBMAKEOPTS) -C test/bpf/
-	test/bpf/unit-test
```

Comment:
Can you please describe the motivation for removing this line?
`test/bpf/unit-test.c` still seems to exist and it also seems to have some tests which are not part of the cmock tests. Or am I missing something?

Reply:
I had suggested this along with changes to `test/bpf/Makefile` that should cause this target to be run automatically.

Reply:
The process to run all the tests is now inside target `unit-tests` in `test/bpf/Makefile`.

Reply:
Ah apologies, I missed the earlier discussion and the `test/bpf/Makefile` part that @joestringer kindly pointed below.